WTG.Analyzers.Test/TestData/FileSystemPathsAnalyzer/AcceptablePlatformSpecificPaths/Source.cs

```diff
@@ -0,0 +1,20 @@
+using System.IO;
+
+class Foo
+{
+	const string ChildPath = "MyCoolDir";
+
+	public static void Method()
+	{
+		DoThingWithFile(Path.Combine("C:\\Blah", "My Cool Dir"));
+		DoThingWithFile(Path.Combine("/usr/bin", "vim"));
```

Comment:
the leading forward-slash, it’s the only indicator I have to go on

Reply:
I can understand the need for the leading slash, because that identifies it as being root-relative. It's the second slash that I was questioning, should we insist it be rewritten as `Path.Combine("/usr", "bin", "vim")`?

Reply:
ah, hmm.

Personally I think it is more readable to have the base of the path in a single block where it is already a valid/correct path, such as `Path.Combine("/usr/local", ProductName, "foo")`. In the reverse scenario (Windows-style) we would otherwise be asking the developer to do something like `Path.Combine("\\?", "C:", "Windows")` which just doesn't read well IMO.

Technically, since we know that this is now using forward slashes, we don't need Path.Combine for correctness and could allow `Path.Combine("/usr/local", "foo/bar")` but that just feels weird to me.

Reply:
ok